README.md

```diff
@@ -39,6 +39,21 @@ pf.animal.search()
     });
 ```
 
+Another way if you have issues with the above code snippet and using React
```

Comment:
This snippet better fits under the TypeScript/ES6 Module section. Do you want to replace that snippet with this one and then update the title to include React?

Reply:
Or we could also create a new Usage (React) section and add this and the jsx under that.

Reply:
I personally would just completely replace the snippet with React example, but if you want to make a separate section for React that would be fine too.

Reply:
React is different than browser, so I would keep it separate

Reply:
sounds good!